src/front/glsl/lex_tests.rs

```diff
@@ -1,5 +1,7 @@
 use super::lex::Lexer;
 use super::parser::Token;
+use super::parser::Token::*;
```

Comment:
nit: generally the style we are trying to keep here is one crate per use statement, i.e. this would be something like:
```rust
use super::{lex::Lexer, parser::Token::{self, *}, token::TokenMetadata};
```
this is not a blocker for merging though:)

Reply:
done